regression-test/data/load_p0/broker_load/simple_array.data

```diff
@@ -0,0 +1,3 @@
+1/[1,2,3,4,5]/[32767,32768,32769]/[65534,65535,65536]/["a","b","c","d","e"]/["hello","world"]/["1991-01-01"]/["1991-01-01 00:00:00"]/[0.33,0.67]/[3.1415926,0.878787878]/[1,1.2,1.3]
+2/[1,2,3,4,5]/[32767,32768,32769]/[65534,65535,65536]/["a","b","c","d","e"]/["hello","world"]/\N/\N/\N/\N/[1,1.2,1.3]
+3/\N/\N/\N/\N/\N/\N/\N/\N/\N/\N
```

Comment:
I think the cases should cover partial `\N`s in one array. E.g. [1, \N, 3]

Reply:
done